Form/editform.py

```diff
@@ -77,7 +77,7 @@ class EditForm(ManagedWindow):
     """
     Form Editor.
     """
-    def __init__(self, dbstate, uistate, track, citation):
+    def __init__(self, dbstate, uistate, track, citation, callback):
```

Comment:
Instead of using a callback, the gramplet should connect to the appropriate database signals.  In this way the gramplet will update if changes are made using the standard editors.

Reply:
I looks to me like it would need to connect to signals for all events, sources and citations as well as the current person (already connected) to catch the case of someone manually adding the right things to make a form appear.

I judged the callback (which API seems to be there for a reason) to be simpler and to cover the common case where someone adds a new form, and would like to actually see it in the list afterword.

It's your code, feel free to accept or change my suggested changes.  I was just trying to help a user.

Reply:
@prculley OK.  It looks like I found a related bug that you didn't attempt to fix.  Some people do edit forms, but are less likely to create new forms, using the standard editors.